po/es_NI.po

```diff
@@ -442,8 +442,10 @@ msgid ""
 msgid_plural ""
 "The earliest transaction date found in this book is %s. Based on the "
 "selection made above, this book will be split into %d books."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "La fecha de transacción más temprana que se encuentra en este libro es% s. Basado en la "
+"selección realizada anteriormente, este libro se dividirá en% d libro."
```

Comment:
There's a spacing error here and in the following line: It should be `en %d`, not `en% d`.

Reply:
libro  --> libros (?)